### ISSUE TITLE
Leased and Unpublished item found in search result of the user

### ISSUE THREAD

**Issue by Karthikeya212** [avalonmediasystem/avalon#1686]
### Descriptive summary

Server: spruce.dlib.indiana.edu
Testing: Regression
Browser: Chrome(56.0.2924.87 (64-bit))
Operating System: Windows10
Issue: An item that is leased to a user until a specified date, and is not published by the manager yet is found in the browse list and search result of the user.

### Steps to reproduce the behavior

1. Open two windows with spruce system, and login with a Manager and a avalon User.
2. Using Manager window, add lease for the user for an Item.
![lease list](https://cloud.githubusercontent.com/assets/25850972/23566551/56e91a1a-0020-11e7-8746-e3cf54afc819.PNG)

3. Check that the video is unpublished.
![unpublished item](https://cloud.githubusercontent.com/assets/25850972/23566554/5b64898a-0020-11e7-877a-f8a5e606188e.PNG)

4. Go to the User window and search for the item.
![lease_unpub item found](https://cloud.githubusercontent.com/assets/25850972/23566566/73d72112-0020-11e7-856b-c4b31f844185.PNG)

### Related work

Link to related tickets or prior related work here.

**Comment by cjcolvar** [comment]
I couldn't reproduce this on spruce.

**Comment by Karthikeya212** [comment]
Issue found again on sodium server.

![access controls](https://user-images.githubusercontent.com/25850972/27349826-903ca862-55c6-11e7-860f-e1f075a74595.PNG)

![item preview](https://user-images.githubusercontent.com/25850972/27349830-928faec0-55c6-11e7-8f4b-e5965955d00d.PNG)

![search result](https://user-images.githubusercontent.com/25850972/27349835-94e94028-55c6-11e7-85b9-bcc642d92860.PNG)

**Comment by cjcolvar** [comment]
@Karthikeya212 It doesn't look like the item is "hidden" from your screenshots.  Could you click the "Hide this item from search results" checkbox and see if it is still showing up?

**Comment by Karthikeya212** [comment]
@cjcolvar yes the item is not hidden, the purpose here is to show that an Item that is unpublished is visible to the user. Once we provide lease to a user, the item should be visible to him only after it is published right?

**Comment by cjcolvar** [comment]
You're right.  Sorry, I didn't read over the whole ticket.

**Comment by cjcolvar** [comment]
I was not able to reproduce this issue of unpublished leases causing a user to be able to see it in search results.  I was however able to reproduce the issue of published leased hidden items being visible in search results.

**Comment by cjcolvar** [comment]
Related to https://bugs.dlib.indiana.edu/browse/VOV-3394

**Comment by davidschober** [comment]
edge case, won't fix.
